src/main/java/com/jalasoft/jfc/model/image/ImageParam.java

```diff
@@ -26,6 +26,9 @@
     // Contains a boolean value for building a Thumbnail.
     private boolean isThumbnail;
 
+    // Contains a boolean value for building a Thumbnail.
```

Comment:
This variable is "isGrayscale" not thumbnail

Reply:
It is done.